pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -387,6 +389,9 @@ def do_update_config(self, state_cache_size_mb):
                 for i in range(4):
                     config['store']['trie_cache']['per_shard_max_bytes'][
                         f's{i}.v1'] = state_cache_size_mb * 10**6
+            if state_snapshot_enabled is not None:
+                key = 'state_snapshot_enabled'
+                config['store'][key] = state_snapshot_enabled
```

Comment:
Note to self: `state_snapshot_compaction_enabled` is a performance improvement applicable in certain contexts, and it's not needed for mocknet.